src/confd/yang/infix-if-bridge@2024-03-06.yang

```diff
@@ -235,22 +238,28 @@ submodule infix-if-bridge {
     }
   }
 
-  grouping mdb {
-    container mdb {
+  grouping multicast-filters {
+    container multicast-filters {
+      presence multicast;
       description "Bridge multicast database.";
-      config false;
 
-      list entry {
-        description "MDB entry.";
+      list multicast-filter {
+        description "Multicast filter entry.";
         key "group";
 
         leaf group {
-          description "IP multicast group.";
-          type rt-types:ip-multicast-group-address;
+          description "L2/L3 multicast address.";
+          type union {
+            type rt-types:ip-multicast-group-address;
+            type yang:phys-address;
+          }
         }
-
         list ports {
           description "Port members of group.";
+          must "not(re-match(../group, '([0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})') and state != 'permanent')" {
+            error-message "If group type is MAC multicast, state must be permanent.";
+          }
+          key "port";
```

Comment:
Setting a static IP multicast entry should also be permanent.  Only the IGMP/MLD code in the kernel sets `temporary`, so this must expression seems redundant.